stories/link-bars-complete-examples/Elbil24Story.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import { DemoContainer } from '../storybook-components';
+
+import {
+	HugeHeading,
+
+	Link,
+	LinkBarLink,
+	HorizontalLinkBar,
+	Paragraph,
+
+	Elbil24Logo,
+	Elbil24ProButton,
+
+	getColor,
+} from '../../src';
+
+const linkProps = {
+	useUnderline: false,
+	activeBackgroundColor: getColor('secondary'),
+	textColor: 'white',
+	ALLCAPS: true,
+};
+
+// const TopNavigation = styled(HorizontalLinkBar)`
+const TopNavigation = styled.div`
+	background: ${getColor('primary')};
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	width: 100%;
+	height: 145px;
```

Comment:
rems are better

Reply:
Will fix things like that when we have final html/css